Mathlib/Topology/Separation.lean

```diff
@@ -2247,37 +2268,54 @@ end Normality
 
 section CompletelyNormal
 
-/-- A topological space `X` is a *completely normal Hausdorff space* if each subspace `s : Set X` is
-a normal Hausdorff space. Equivalently, `X` is a `T₁` space and for any two sets `s`, `t` such that
+/-- A topological space `X` is a *completely normal space* if each subspace `s : Set X` is
+a normal space. Equivalently, for any two sets `s`, `t` such that
 `closure s` is disjoint with `t` and `s` is disjoint with `closure t`, there exist disjoint
 neighbourhoods of `s` and `t`. -/
```

Comment:
Could you please use the opportunity to say in the docstring what are the lemmas that give the facts you quote? The current docstring mentions two facts and none of them is exactly the definition used in this declaration.